### DISCUSSION TITLE
Directory Opus 和 OneCommander 这两款著名的文件管理器，用了几天，使用感受

### DISCUSSION THREAD

**Discussion by Chaoses-Ib** [Chaoses-Ib/DirectoryOpus#7] [讨论]
@💤:

> Directory Opus 和 OneCommander 这两款著名的文件管理器，用了几天，使用感受。
> ① Directory Opus 设置选项 自定义太多，密密麻麻，看的头晕，感觉专为“码农”设计，强迫症最爱。而 OneCommander 设置选项少，主流选项，描述也易懂。
> 
> ② Directory Opus 界面比 OneCommander 难看（虽然可以自定义），而且，界面排版也不如 OneCommander。OneCommander 界面太漂亮！！
> 
> ③ Directory Opus 操作比 OneCommander 流畅，OneCommander 打开和预览文件夹和图片，明显感知卡一些。（感觉 OneCommander 还没有全力优化64位和多核多线程）。
> 
> ④ Directory Opus 自研传输界面和技术，而 OneCommander 调用系统自带，实测：已传输500GB文件，Directory Opus 传输速度和稳定性 明显比 OneCommander 好太多，PS:原来系统自带传输很垃圾！！
> 
> ⑤ 两款软件，都有设置成  系统默认管理器 这个功能，但是 Directory Opus比  OneCommander 人性化和稳定。
> 
> ⑥ 两款软件，自带 压缩 和 看图 功能，Directory Opus 比 OneCommander 好用太多，（虽然我全用第三方）

**Comment by Silent-wqh** [comment]
我用了小一年的 OneCommander，最近重装系统后尝试了 Directory Opus。最明显的感受是速度，Directory Opus 的启动速度明显快很多。我使用 OneCommander 时，启动第一个窗口时总觉得很慢，让我很不舒服。

不过，OneCommander 的界面确实更友好，也有透明特效，但我个人并不在意这些，平时也是关闭的。现在使用 Directory Opus 感觉非常顺畅，短期内我不会再回到 OneCommander 了。

**Reply by Chaoses-Ib** [reply]
DO 借助些像 [MicaForEveryone](https://github.com/MicaForEveryone/MicaForEveryone) 的第三方工具也能实现透明效果，虽然也可能会导致 bug：

![image](https://github.com/user-attachments/assets/ead070b3-98f6-4007-a939-7fc45d764cea)

[Dolphin Theme (with Mica effect)](https://resource.dopus.com/t/dolphin-theme-with-mica-effect/42323?u=chaoses-ib)

**Comment by Chaoses-Ib** [comment]
- DOpus 支持压缩包 VFS，OneCommander 不支持
